.github/workflows/ci.yml

```diff
@@ -108,6 +108,16 @@ jobs:
       - name: test changed packages
         run: yarn backstage-cli repo test --coverage --maxWorkers=3
 
+      - name: install playwright
+        if: ${{ hashFiles(format('workspaces/{0}/playwright.config.ts', matrix.workspace)) != '' }}
+        run: npx playwright install --with-deps chromium chrome
+
+      - name: run playwright tests
+        if: ${{ hashFiles(format('workspaces/{0}/playwright.config.ts', matrix.workspace)) != '' }}
+        run: npx playwright test
+        env:
+          DEVMODE: true
```

Comment:
Why do we need another env here? Isn't CI good enough and DEVMODE is something that someone might use locally?

Reply:
I could use some opinions here. The status quo in playwright configs is that `CI` disables dev mode. I'm trying to run dev mode on CI. Should I just flip the condition then?

Reply:
I have a feeling you'll like the new version more (since it was your idea) :laughing: